yarn.lock

```diff
@@ -19226,6 +19226,11 @@ search-insights@1.7.1:
   resolved "https://registry.yarnpkg.com/search-insights/-/search-insights-1.7.1.tgz#eddfa56910e28cbbb0df80aec2ab8acf0a86cb6b"
   integrity sha512-CSuSKIJp+WcSwYrD9GgIt1e3xmI85uyAefC4/KYGgtvNEm6rt4kBGilhVRmTJXxRE2W1JknvP598Q7SMhm7qKA==
 
+search-insights@2.3.0:
+  version "2.3.0"
+  resolved "https://registry.yarnpkg.com/search-insights/-/search-insights-2.3.0.tgz#9a7bb25428fc7f003bafdb5638e90276113daae6"
+  integrity sha512-0v/TTO4fbd6I91sFBK/e2zNfD0f51A+fMoYNkMplmR77NpThUye/7gIxNoJ3LejKpZH6Z2KNBIpxxFmDKj10Yw==
+
```

Comment:
Is it possible to have a single `search-insights` dependency version in the monorepo?

Reply:
You mean use the same version between examples and the peer dependency in `@algolia/autocomplete-plugin-algolia-insights`?

Reply:
I'll do this in a separate PR if that's okay.

For the examples, the easiest would be to directly use v3 once it's available so we don't have to add a cookie banner and still have those examples work.

Regarding `search-insights@^2.1.0` this is a dependency coming from `instantsearch.js` which we'll have to update as well, so I'd love to keep this rabbit hole independent from this change :D